xAPI.md

```diff
@@ -566,33 +589,30 @@ and so the LRS can return this list of agents in any order. See [4.1.2.2 Groups]
 * Attachments. These are not part of statements and an LRS will return statements without attachments when the client
 requests this (see the [Statement API's](#stmtapi) "attachments" parameter for details).
 
+The following explictly are **not** exceptions and **are** covered by this rule:
```

Comment:
This section is new and allows us to explicitly call out duration as not something you can deserialise.